docs/faq.md

```diff
@@ -114,3 +114,25 @@ For the Example project, run the `EarlGreyExampleSwiftTests` target once then fi
 For physical device builds, replace `Debug-iphonesimulator` with `Debug-iphoneos`.
 
 You can verify the `EarlGrey.framework` is included in the bundle at `EarlGreyExampleSwiftTests.xctest/Frameworks/EarlGrey.framework`
+
+**How should I handle animations?**
+
+Infinite animations must be disabled as they'll cause problems with EarlGrey tests.
```

Comment:
Forgot to mention:
EarlGrey has a way to force cyclic animations to finish sooner using kGREYConfigKeyCALayerMaxAnimationDuration config key.

```
[[GREYConfiguration sharedInstance] setValue:@(kMaxAnimationInterval)
                                forConfigKey:kGREYConfigKeyCALayerMaxAnimationDuration];
```